src/index.ts

```diff
@@ -1,3 +1,6 @@
 // Activities will be re-exported from this file.
 
+export { default as AddImageToPdf } from "./activities/AddImageToPdf";
```

Comment:
I'm a bit confused about the `action` for these. The `uuid.js` file has a regular auto-generated UUID prefix, but it looks like this PR implements these activities:
* `gcx:wf:pdf::AddImageToPdf`
* `gcx:wf:pdf::AddTextToPdf`
* `gcx:wf:pdf::CreatePdf`
Is it better to have two separate sets of actions?

Reply:
This activity pack produces actions like `uuid:2f9f2575-a722-4f4d-9b61-4e5452b11eaf::AddImageToPdf`.

In `workflow-ts` we defined actions like `gcx:wf:pdf::AddImageToPdf` in the stubs for the productized .NET implementations.

This difference is intended. This activity pack is not in the product so it must use difference actions. One day we may decide to productize a TypeScript version of these activities. At that point we would deprecate the activity pack and move its implementations into the `workflow-ts` stubs where they would use the `gcx:wf:pdf::` actions.